packages/backend/init-action/main.js

```diff
@@ -37,10 +38,22 @@ const run = async () => {
     }),
   });
 
+  const reponseBuilder = {
+    meta: {
+      repo_name: GIT_REPO,
+      repo_owner: GIT_OWNER,
+      start_time: getCurrentTimestamp(),
+      commit_sha: GITHUB_SHA,
+    },
+    history: history,
```

Comment:
We _could_ make this quicker, by not _only_ having the objects (that need to be looped through) - but also an _index_ object. 

```js
const reponseBuilder = {
  meta: {
    repo_name: GIT_REPO,
    repo_owner: GIT_OWNER,
    start_time: getCurrentTimestamp(),
    commit_sha: GITHUB_SHA
  },
  historyIndex: {
    "1": 0,
    "2": 1,
    "1232321": 2,
    "123212u9ydsadsa": 3
  },
  history: [
    {
      sha: "1",
      data: "foo bar baz"
    },
    {
      sha: "2",
      data: "foo bar baz"
    },
    {
      sha: "1232321",
      data: "foo bar baz"
    },
    {
      sha: "123212u9ydsadsa",
      data: "foo bar baz"
    }
  ]
};

```

Reply:
Means you can go 

```js
const index = reponseBuilder.historyIndex[commit.sha];
const commit = reponseBuilder.history[index]
```

Which means there's no looping and looking for the commits.